src/Base/Bitmask.h

```diff
@@ -113,7 +113,7 @@ class Flags {
     Enum i;
 
 public:
-    constexpr inline Flags(Enum f) : i(f) {}
+    constexpr inline Flags(Enum f = Enum()) : i(f) {}
```

Comment:
I believe the linter is wrong in this case, that this one cannot be explicit (or if we choose to make it so, we're going to have to add some casts elsewhere).

Reply:
Probably would be wise to put // NO-LINT runtime/explicit here with a comment why it is ok.